libs/platform/src/lib/approval-flow/approval-flow.component.ts

```diff
@@ -413,15 +474,17 @@ export class ApprovalFlowComponent implements OnInit, OnDestroy {
 
     /** @hidden Fetch all necessary data and enter edit mode */
     _enterEditMode(): void {
-        this._editModeInitSub = this.dataSource
-            .fetchWatchers()
-            .pipe(take(1))
-            .subscribe((watchers) => {
-                this._usersForWatchersList = watchers;
-                this._selectedWatchers = [...this._approvalProcess.watchers];
-                this._isEditMode = true;
-                this._initialApprovalProcess = cloneApprovalProcess(this._approvalProcess);
-            });
+        // there's no support for searching in multi-input, so grabbing all watchers
+        // triggering initial loading of data in data sources
+        this.watcherDataSource.match();
+        this._editModeInitSub = this.watcherDataSource.open().subscribe((watchers) => {
```

Comment:
in some places I see dataSource.close() approach but here we don't.
Is it intentional?

Reply:
missed this, thanks. added now